### PR TITLE
 framework/wifi_manager: Arrange SSID and Passphrase length

### PR DESCRIPTION
**Pull request by SeonghoByeon**
- Set the SSID and Passphrase string length to 33 and 65, respectively
- Insert null after strncpy, only when the string is not initialized (to null)
- In case WiFi Manager recieves callback msg (such as scanning result), check the SSID length not to be larger than 32
- Change SSID and Passphrase length condition in wm_test and tcp_tls_stress (commit 95317f5)

### CODE REVIEW

**Comment by SeonghoByeon** [comment]
Hi, @sunghan-chang, @JeongJunSik, and @pillip8282.
as mentioned before, we set the length of SSID and PASSPHRASE as 33 and 65 bytes strings, respectively, and accordingly, modify all the conditions in the entire code. Please review and let me know if any correction is required.

**Comment by seinfra** [comment]
Target : [95317f5771d2be10a3dd3fa70eeae7104347fa8b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [95317f5771d2be10a3dd3fa70eeae7104347fa8b] - Code Rule Check (C++) OK.

File: apps/examples/testcase/le_tc/tcp_tls/tcp_tls_stress.c [hunk 1]
```diff
@@ -212,13 +212,13 @@ tcp_tls_result_e tcp_tls_config_get(tls_config * config)
 
 static void wm_get_apinfo(wifi_manager_ap_config_s *apconfig)
 {
-	strncpy(apconfig->ssid, WM_AP_SSID, WIFIMGR_SSID_LEN-1);
-	apconfig->ssid[WIFIMGR_SSID_LEN-1] = '\0';
+	strncpy(apconfig->ssid, WM_AP_SSID, WIFIMGR_SSID_LEN);
```
Comment (jsdosa) [inline apps/examples/testcase/le_tc/tcp_tls/tcp_tls_stress.c]: should check the size of WM_AP_SSID and WIFIMGR_SSID_LEN.

File: framework/src/wifi_manager/wifi_manager.c [hunk 2]
```diff
@@ -1372,8 +1381,8 @@ wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info)
 	LOCK_WIFIMGR;
 	uint8_t *ip = (uint8_t *)&g_manager_info.ip4_address;
 	snprintf(info->ip4_address, 18, "%d.%d.%d.%d", ip[0], ip[1], ip[2], ip[3]);
-	memcpy(info->ssid, g_manager_info.ssid, 32);
-	memcpy(info->mac_address, g_manager_info.mac_address, 6);
+	memcpy(info->ssid, g_manager_info.ssid, WIFIMGR_SSID_LEN + 1);
```
Comment (JeongJunSik) [inline framework/src/wifi_manager/wifi_manager.c]: "WIFIMGR_SSID_LEN + 1" or "WIFIMGR_MACADDR_LEN"

Comment (SeonghoByeon) [reply framework/src/wifi_manager/wifi_manager.c]: Here, SSID (info->ssid) should be the length of WIFIMGR_SSID_LEN + 1.
And as below, MAC addr (info->mac_address) should be the length of WIFIMGR_MACADDR_LEN.

File: apps/examples/testcase/le_tc/tcp_tls/tcp_tls_stress.c (hunk 1 above)
Comment (SeonghoByeon) [reply apps/examples/testcase/le_tc/tcp_tls/tcp_tls_stress.c]: That is true. Thanks for the comment. Modification completed.

**Comment by seinfra** [comment]
Target : [d4eb08c39d4c1b76b54a714f452174bb00c10835] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d4eb08c39d4c1b76b54a714f452174bb00c10835] - Code Rule Check (C++) OK.

**Comment by SeonghoByeon** [comment]
Hi @intom, thanks for the commet. Because the length of WM_AP_SSID is shorter than (or equal to) the size of apconfig->ssid, remaining space of apconfig->ssid could be filled with unknown value. In my opinion, since the length of WM_AP_SSID is variable depending on the configuration, it is correct to copy it as much as the size of its length.

**Comment by seinfra** [comment]
Target : [42eda3f04bfc88e3304e8c4e70d30481ea46bf61] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [42eda3f04bfc88e3304e8c4e70d30481ea46bf61] - Code Rule Check Result:
framework/src/wifi_manager/wifi_manager.c:549: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/wifi_manager/wifi_manager.c:549: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/wifi_manager/wifi_manager.c:549: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

**Comment by seinfra** [comment]
Target : [9c67ebd7eb3a1a057f0f2a1427ffd318253ef8dc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9c67ebd7eb3a1a057f0f2a1427ffd318253ef8dc] - Code Rule Check Result:
framework/src/wifi_manager/wifi_manager.c:549: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/wifi_manager/wifi_manager.c:549: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/wifi_manager/wifi_manager.c:549: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

**Pull request closed without merging**